sdk/test/lib/redeemer-proxy.test.ts

```diff
@@ -120,8 +124,9 @@ describe("OrangeKitTbtcRedeemerProxy", () => {
     it("the sign message function passed to the orange kit should call the bitcoin provider", async () => {
       const signMessageFn = spyOnSendTransaction.mock.calls[0][5]
       const mockedMessage = "test"
+      const mockedSafeTxData = { to: "test" } as SafeTransactionData
 
-      await signMessageFn(mockedMessage)
+      await signMessageFn(mockedMessage, mockedSafeTxData)
```

Comment:
Can we add tests that cover a provider that implements `signWithdrawMessage` and the one that doesn't?

Reply:
7ff32cb